index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <header>
+    <div class="logo">Bright Ezeamaka</div>
+    <div class="dropdown">
+      <img src="./Images/Icon - Menu.png" alt="">
+    </div>
```

Comment:
- [ ] While you did a wonderful job adding the Hamburger button :100:, it is required to wrap it with a `nav` tag for semantics reasons :+1:, so instead using `<div>` you should use `<nav>`